### ISSUE TITLE
visualize historical test results

### ISSUE THREAD

**Issue by RomanHauksson** [coloradocolby/thokr#11]
## Problem

There's no way to visualize historical test data, such as WPM and accuracy.

## Solution

I would love to help write some code (in Python, if you're alright with that) to generate graphs that show average WPM and accuracy over time, for example. The logs would have to record the game mode for this to work. Let me know what you think.

## Alternatives

A simple statistics summary page could show one's PB for each game mode, total time spent typing, etc.

## Additional context

*Statistics graph in Monkeytype.*

![Thokr_graph](https://user-images.githubusercontent.com/62364808/166780397-a61add82-e67d-4f14-8a11-baae43f56106.png)

**Comment by coloradocolby** [comment]
Interesting idea! Let me think on this a bit more – my initial reaction is that I'd love to keep everything in rust, but I do think the general idea of consuming the logged results for something would be really neat! I'd be curious to see how well a graph like the one you posted above could actually be rendered in the terminal 🤔

**Comment by RomanHN** [comment]
Yeah, that's probably the main problem. I really like the braille-rendered graph at the end of each test, but it could also just generate a PNG graph in `.config/thokr/`.
